string episode_id
string waypoint
string meta_data
uint32 timestamp

sensor_msgs/PointCloud2 cloud
sensor_msgs/Image rgb_img
sensor_msgs/Image depth_img

string camera_info_topic
string robot_pose_topic
---
bool result
soma_llsd_msgs/Scene response
